_data/simple-icons.json

```diff
@@ -1285,6 +1285,11 @@
             "hex": "1EABE2",
             "source": "https://cevo.com/"
         },
+        {
+            "title": "Chainlink",
+            "hex": "2A5ADA",
+            "source": "https://chain.link"
```

Comment:
Icon looks spot on to me - is it worth linking directly to the [brand assets](https://chain.link/brand-assets) page? That contains more info about the colours etc, so would be more helpful to users I feel.

Reply:
Updated